www/js/diary/services.js

```diff
@@ -144,26 +174,33 @@ angular.module('emission.main.diary.services', ['emission.plugin.logger',
       return background;
   }
 
-  dh.getLocalTimeString = function (dt) {
+  dh.getLocalTimeString = function (dt, includeDay=false) {
+    if (!dt) return;
```

Comment:
I don't like adding on the day to what is supposed to be the time string. We should handle this in the HTML. If there is a reason why that won't work/is awkward, please document it.

Reply:
Where do we even pass in `includeDay=true`? The only two calls seem to be around here 
https://github.com/e-mission/e-mission-phone/pull/962/files#diff-3692dc7185f771351204f053be118ba7122c58fc2a51f2c0d50ae838e3a97351R436

and neither of them seem to pass in this argument.

Reply:
What am I missing here?

Reply:
You're right, I can take out that unused parameter.
it's because I originally used this option to just append the date, but then decided to refactor that functionality into `getDisplayDateAbbr()` and store it in separate fields.